snippets/javascript.json

```diff
@@ -41,37 +33,32 @@
     "prefix": "hpext",
     "description": "Hapi.js Server Extension",
     "body": [
-      "server.ext('${event}', (request, reply) => {",
-      "\t${1}",
-      "\treturn reply.continue();",
-      "});"
-    ]
-  },
-  "server handler": {
-    "prefix": "hphandler",
-    "description": "Hapi.js Server Handler",
-    "body": [
-      "server.handler('${name}', (route, options) => {",
-      "\t${1}",
+      "server.ext({",
+      "\ttype: '${event}',",
+      "\tmethod: (request, reply) => {",
```

Comment:
the documentation is consistent using `h` instead of `reply` as variable name, and it always uses `function` instead of `lambda`. 

```
server.ext({
        type: 'onRequest',
        method: function (request, h) {

            // Change all requests to '/test'

            request.setUrl('/test');
            return h.continue;
        }
    });
```